pom.xml

```diff
@@ -42,17 +42,17 @@
     <java.minimum.version>1.8</java.minimum.version>
   </properties>
 
-  <!-- This configures the publication of jars to the S3 bucket by default. -->
+  <!-- This configures the publication of jars to GoDaddy Artifactory by default. -->
```

Comment:
Question: should we also configure the Artifactory repository for reads below in the `repositories` list?

Reply:
Definitely! `artifactory.mirror` is the id for read credentials.